src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((declaration) => declaration.split(':'))
+    .filter(([property, value]) => property && value)
```

Comment:
The `filter` function is used to remove any entries where either the property or value is missing. This is a good practice to prevent adding undefined properties to the resulting object.